test/travis/send_receive/setup.go

```diff
@@ -0,0 +1,100 @@
+package send_receive
+
+import (
+	"github.com/interconnectedcloud/qdr-operator/pkg/apis/interconnectedcloud/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rh-messaging/qdr-shipshape/pkg/messaging"
+	"github.com/rh-messaging/shipshape/pkg/apps/qdrouterd/deployment"
+	"github.com/rh-messaging/shipshape/pkg/apps/qdrouterd/qdrmanagement"
+	"github.com/rh-messaging/shipshape/pkg/apps/qdrouterd/qdrmanagement/entities"
+	"github.com/rh-messaging/shipshape/pkg/framework"
+	"github.com/rh-messaging/shipshape/pkg/framework/operators"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+const (
+	IcInteriorRouterName = "interior"
+	ConfigMapName        = "messaging-files"
+)
+
+var IcInteriorRouterSpec *v1alpha1.InterconnectSpec = &v1alpha1.InterconnectSpec{
+	DeploymentPlan: v1alpha1.DeploymentPlanType{
+		Size:      1,
+		Image:     "quay.io/interconnectedcloud/qdrouterd:latest",
+		Role:      "interior",
+		Placement: "Any",
+	},
+}
+
+var (
+	travisFramework *framework.Framework
+	ConfigMap       *v1.ConfigMap
+)
+
+var _ = BeforeEach(func() {
+	// Initializes using only Qdr Operator
+	travisFramework = framework.NewFrameworkBuilder("travis").WithBuilders(operators.SupportedOperators[operators.OperatorTypeQdr]).Build()
+
+	// Generates a config map with messaging files (content) to be
+	// used by the AMQP QE Clients
+	generateMessagingFilesConfigMap()
+
+	deployInterconnect()
+	By("Validating router created")
+	time.Sleep(5 * time.Second) //wait for router to start
+	validateNetwork()
+})
+
+func generateMessagingFilesConfigMap() {
+	var err error
+	ctx := travisFramework.GetFirstContext()
+	ConfigMap, err = ctx.Clients.KubeClient.CoreV1().ConfigMaps(ctx.Namespace).Create(&v1.ConfigMap{
+		ObjectMeta: v12.ObjectMeta{
+			Name: ConfigMapName,
+		},
+		Data: map[string]string{
+			"small-message.txt": messaging.GenerateMessageContent("ThisIsARepeatableMessage", 1024),
+		},
+	})
```

Comment:
@nicob87 maybe use the "GenerateSmallMediumLargeMessagesConfigMap" which is available in the pkg instead?

Reply:
done.